homeassistant/components/template/number.py

```diff
@@ -118,34 +104,24 @@ class TemplateNumber(TemplateEntity, NumberEntity):
     def __init__(
         self,
         hass: HomeAssistant,
-        name_template: Template,
-        value_template: Template,
-        availability_template: Template | None,
-        command_set_value: dict[str, Any],
-        step_template: Template,
-        minimum_template: Template | None,
-        maximum_template: Template | None,
-        optimistic: bool,
+        config,
```

Comment:
Could a type hint be added (maybe also in the parent class `TemplateEntity`)?
```suggestion
        config: dict[str, Any],
```

Reply:
Yes, in another set of PRs typing the template platforms👍